ceph-iscsi.spec

```diff
@@ -67,6 +68,7 @@ Requires:       python3-rtslib >= 2.1.fb68
 Requires:       python3-cryptography
 Requires:       python3-pyOpenSSL
 Requires:       python3-requests
+Requires:       python3-distro
```

Comment:
Nit: Put it in the same spot as the one for legacy Python (i.e. between `python3-setuptools` and `python3-rados` for consistency.

Reply:
Done.